### ISSUE TITLE
About the image requirements.

### ISSUE THREAD

**Issue by itemx** [pkhungurn/talking-head-anime-demo#25]
I am using conda to create an environements on Windows 7. The example waifu works nice, but while trying to use some cropped/moded PNG files from photoshop, the result become weired.

Is that because something unable to determine from eyes in the file polluted my self-prepared PNG picture?

![Result](https://user-images.githubusercontent.com/18457683/107119446-92c10700-68c2-11eb-8a09-db01d7458ec3.png)
![My Sample File](https://user-images.githubusercontent.com/18457683/107119456-a2d8e680-68c2-11eb-98a1-b0a3318b184c.png)

Thanks for the efforts and the time you spent on the project.

**Comment by dragonmeteor** [comment]
Please see https://github.com/pkhungurn/talking-head-anime-demo/issues/1

TL;DR: Use GIMP to export PNG. When you do so, uncheck "save color values from transparent pixel."
